MessengerOnFileForLinux/ChatControl/test/UserInactivityDetectorTestCore.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include "UserInactivityDetector.hpp"
+
+namespace
+{
+std::string getUser()
+{
+    std::string user = getenv("USER");
+    if(user == "ciGitlab")
+    {
+        return "root";
+    }
+    else
+    {
+        return user;
```

Comment:
return getenv("USER") i usunąc linijkę 11

Reply:
Ogólnie całą funkcję bym zrobił jako : ? jest na tyle nieważna, że tylko zasłania podgląd klasy

Reply:
Niestety nie wiem czemu, ale bez przypisania w linijce 11 nie chce to działać i lecą empty_string_ctor exception w UTekach jak się zwraca bezposrednio getenv. 

Zamienione na ? :